lib/v2/bloomberg/utils.js

```diff
@@ -0,0 +1,289 @@
+                        pubDate: parseDate(episode.publishedAt) || item.pubDate,
+                        author: audio_json.hero.showTitle,
+                        media: {
+                            content: { url: episode.image },
+                            thumbnails: { url: episode.image },
+                        },
+                        enclosure_type: 'audio/mpeg',
+                        enclosure_url: episode.url,
+                        itunes_item_image: episode.image || audio_json.pageInfo.image.url,
+                    };
+                    return rss_item;
+                } else if (page === 'videos') {
+                    const json = cheerio.load(res.data)(api.sel).text().trim().replace('window.__PRELOADED_STATE__ = ', '').slice(0, -1);
+                    const article_json = JSON.parse(json);
+                    const video_story = article_json.video?.videoStory ?? article_json.quicktakeVideo?.videoStory;
+                    if (video_story && video_story.video) {
+                        const desc = await processVideo(video_story.video.bmmrId, video_story.summary.html.replaceAll(emptyRegex, ''));
+                        const rss_item = {
+                            title: video_story.headline.text || item.title,
+                            link: video_story.url || item.link,
+                            guid: `bloomberg:${video_story.id}`,
+                            description: art(path.join(__dirname, 'templates/video_media.art'), desc),
+                            pubDate: parseDate(video_story.publishedAt) || item.pubDate,
+                            media: {
+                                content: { url: video_story.video.thumbnail.url },
+                                thumbnails: { url: video_story.video.thumbnail.url },
+                            },
+                            category: desc.keywords ?? [],
+                        };
+                        return rss_item;
+                    }
+                } else if (page === 'photo-essays') {
+                    const $ = cheerio.load(res.data.html);
+                    const article_json = $(api.sel)
+                        .toArray()
+                        .map((e) => JSON.parse($(e).html()))
+                        .reduce((pv, cv) => ({ ...pv, ...cv }), {});
+                    const rss_item = {
+                        title: article_json.headline || item.title,
+                        link: article_json.canonical || item.link,
+                        guid: `bloomberg:${article_json.id}`,
+                        description: (await processBody(article_json.body, article_json)).replaceAll(emptyRegex, ''),
+                        pubDate: item.pubDate,
+                        author: article_json.authors?.map((a) => a.name).join(', ') ?? [],
+                    };
+                    return rss_item;
+                } else {
+                    const article_json = JSON.parse(
```

Comment:
You can check if `res.redirectUrls` contains `https://www.bloomberg.com/tosv2.html...` here and inside `processVideo` method